test/tools/libtesteth/ImportTest.cpp

```diff
@@ -645,9 +645,18 @@ void ImportTest::checkGeneralTestSectionSearch(json_spirit::mObject const& _expe
 					cerr << trInfo << std::endl;
 					_errorTransactions.push_back(i);
 				}
-			}			
+			}
 			else if (_expects.count("hash"))
-				BOOST_CHECK_MESSAGE(_expects.at("hash").get_str() == toHexPrefixed(tr.postState.rootHash().asBytes()), TestOutputHelper::testName() + " Expected another postState hash! expected: " + _expects.at("hash").get_str() + " actual: " + toHex(tr.postState.rootHash().asBytes()) + " in " + trInfo);
+			{
+				//checking filled state test against client
+				BOOST_CHECK_MESSAGE(_expects.at("hash").get_str() == toHexPrefixed(tr.postState.rootHash().asBytes()),
+									TestOutputHelper::testName() + " on " + test::netIdToString(tr.netId) + ": Expected another postState hash! expected: " + _expects.at("hash").get_str() + " actual: " + toHex(tr.postState.rootHash().asBytes()) + " in " + trInfo);
+				if (_expects.count("logs"))
```

Comment:
I see, there should be an error message when there is `"logs"` but there is no `"hash"`.  Otherwise, the `"logs"` section in fillers will be silently ignored.